playbook/app/pb_kits/playbook/pb_file_upload/file_upload.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_tag("div",
+    aria: object.aria,
+    class: object.classname,
+    data: object.data,
+    id: object.id) do %>
+      <%= pb_rails("form_group", props: {cursor: "pointer"}) do %>
+        <label for="upload" class="pb_button_kit_secondary_inline_enabled label"><%= "#{object.label}" %></label>
+        <%= pb_rails("text_input", props: {
+          type: "file",
+          input_options: {
+            id: "upload", 
+            classname: "cursor_pointer",
+          }
+        }) %>
```

Comment:
Can we also add two things for flexibility?

- input_options as a prop
- BYOI (enable children)